docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+| source_name         | string                | name of the source table                                                                                                                                                                                                         | required               | product                                                                                                                        |
+| target_name         | string                | name of the target table                                                                                                                                                                                                         | required               | product                                                                                                                        |
+| join_columns        | list[string]          | list of column names or column name which acts as primary key to the table                                                                                                                                                       | optional(default=None) | ["product_id"] or ["product_id","order_id"]                                                                                    |
+| jdbc_reader_options | string                | jdbc_reader_option,that helps to parallelize the data read from jdbc sources based on the given configuration.For more info [jdbc_reader_options](#jdbc_reader_options)                                                          | optional(default=None) | "jdbc_reader_options": {"number_partitions": 10,"partition_column": "s_suppkey","upper_bound": "10000000","lower_bound": "10"} |
+| select_columns      | list[string]          | list of columns to be considered for the reconciliation process                                                                                                                                                                  | optional(default=None) | ["id","name","address"]                                                                                                        |
+| drop_columns        | list[string]          | list of columns to be eliminated from the reconciliation process                                                                                                                                                                 | optional(default=None) | ["comment"]                                                                                                                    |
```

Comment:
Should we add
list of columns to be eliminated from the reconciliation process `as in source table `